Added Work section

dariamine2000 opened the pull request:
I've Added all Work section elements and fixed the StyleInt errors

dariamine2000 commented:
Added about Section!

Maclenn77 reviewed (commented):
Hi @dariamine2000 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't load the CSS file. The reason appears to be that in ln 7 you added a slash ` / ` to the path to the file.

In your code it appears:

`  <link rel="stylesheet" href="style.css">`

Try with:

`   <link rel="stylesheet" href="style.css"> ` 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL] Improve your Readme file with:
- A brief description of the project, 
- The tech stack used for its development, 
- A mention of the Figma Template that you're using for your project.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

dariamine2000 commented:
I've Added all Work section elements using both Flexbox and grid, also and Added About me section, and I used template number 1 as an example for my Portfolio.

medaminedev66 commented on style.css:
```diff
@@ -101,3 +120,139 @@ body {
+  border-radius: 8px;
+}
+
+.item img {
+  grid-area: 1/1;
+}
```
- [ ] It would be nicer to center the image in the project card to improve the UI design.

<img width="155" alt="image" src="https://user-images.githubusercontent.com/77190203/164868488-22d8028a-eca7-401a-866d-7db0947c23ee.png">

medaminedev66 commented on style.css:
```diff
@@ -101,3 +120,139 @@ body {
+  grid-area: 4/1;
+  text-align: left;
+}
+
+/* Work List html css javaScript */
+.Work-List {
+  margin-top: 20px;
+  margin-left: 15px;
+  padding-left: 0%;
+  padding-right: 75px;
+  padding-top: 0%;
+  padding-bottom: 0%;
+  height: fit-content;
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+}
+
+.Work-List li {
+  /* Auto layout */
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: -15px;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  position: static;
+  width: fit-content;
+
+  /* Button / Small */
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+
+  /* Primary/B400 */
+  color: #6070ff;
+}
```
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided.
- Kindly add gaps between project cards.
- Try to add paddings to each project card.
- Try to add paddings to the buttons.

medaminedev66 commented on style.css:
```diff
@@ -101,3 +120,139 @@ body {
 }
+
+/* Work section */
+.Work {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
```
- [ ] Following the project requirements, The main container of the projects should have `display:grid` instead `display: flex;`. Kindly work on that.

medaminedev66 commented on index.html:
```diff
@@ -28,5 +29,93 @@ <h1>I’m Amine Glade to see you!</h1>
+        <li><a href="#"><img src="/Icons/angellist.png" alt="angellist"></a></li>
+        <li><a href="#"><img src="/Icons/medium.png" alt="medium"></a></li>
+      </ul>
+      <h3>Get my resume</h3>
```
- [ ] You haven't implemented all the parts of the `About` section. Please add the languages, frameworks, and skills to the `About` section.

medaminedev66 commented on index.html:
```diff
@@ -28,5 +29,93 @@ <h1>I’m Amine Glade to see you!</h1>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <h3>See Project</h3>
```
- [ ] The button (e.g., "See project") should be a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed. Kindly work on that.

medaminedev66 reviewed (changes requested):
Hi @dariamine2000  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Git/GitHub flow best practices have been followed ✔️
- There are no linter errors ✔️


## Required Changes ♻️
- [ ] It will be nice if you could work more on the README file so it is more descriptive and organized. That file will give visitors a brief understanding of your project. Please use the [Microver template](https://github.com/microverseinc/readme-template) as an example. 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
